### DISCUSSION TITLE
`create-quasar` customization

### DISCUSSION THREAD

**Discussion by bazuka5801** [quasarframework/quasar#15621] [General - Components / Directives / etc]
Hello, I want to add my template to the create-quasar command, how do I do this and how do I check it?
P.s. Template for mobile apps. QuasarCapacitor/QuasarCapacitor#1

**Comment by dongwa** [comment]
May be you can find answer in [quasar-legacy-create#scaffolding-from-a-custom-starter-kit](https://quasar.dev/quasar-cli-vite/commands-list#scaffolding-from-a-custom-starter-kit)

**Reply by bazuka5801** [reply]
it's legacy

**Comment by suleimanas** [comment]
better to use App Extensions https://quasar.dev/app-extensions/introduction

> App Extensions replace the need to create custom starter kits. Anything you add on top of the official starter kit can come in the form of an App Extension. This also eliminates maintenance overhead of keeping unofficial starter kits up to date with the official one and so we can ensure that every developer will always get the latest and greatest Quasar specs out of the box.

**Reply by bazuka5801** [reply]
I have created several extensions for quasar, but maintaining the extension is very difficult. This is not a solution, I look at [create-quasar](https://github.com/quasarframework/quasar/tree/dev/create-quasar) over and over again. I appreciate this template code and want to use it to create my own template using this template system, but I can't find how to use my own template from another repository.
